ui/admin/app/abilities/credential.js

```diff
@@ -0,0 +1,14 @@
+import CredentialAbility from 'api/abilities/credential';
+import { inject as service } from '@ember/service';
+
+export default class OverrideCredentialAbility extends CredentialAbility {
+  // =service
+  @service features;
+
+  get canRead() {
+    return this.features.isEnabled('json-credentials') ||
+      this.model.type !== 'json'
+      ? super.canRead
```

Comment:
Shouldn't this be:
```suggestion
      ? super.canRead()
```

Do we have tests for these different scenarios?

Reply:
`canRead` is a getter so you don't invoke it like a function. I actually added the `()` at first and it was causing errors.

Reply:
As for the test question, I asked the team about that. I will update this accordingly.

Reply:
Good point, totally thought it was a function and not a getter